forge-gui/res/cardsfolder/upcoming/druidic_ritual.txt

```diff
@@ -0,0 +1,8 @@
+Name:Druidic Ritual
+ManaCost:2 G
+Types:Sorcery
+A:SP$ Mill | NumCards$ 3 | Optional$ True | SubAbility$ DBReturn | SpellDescription$ You may mill three cards.
+SVar:DBReturn:DB$ ChangeZone | Origin$ Graveyard | Destination$ Hand | Hidden$ True | ChangeType$ Creature.YouOwn | ChangeTypeDesc$ creature card they own | SubAbility$ DBReturn2 | SpellDescription$ Then return up to one creature card
```

Comment:
I suppose they should change zone together, will matter for stuff like _Fuming Effigy_

Reply:
Nice – there are a few other cards that need refactoring in this way too - some of them target so it might be a larger fix in the engine
Crop Sigil, Pull from the Deep, Retrieve, Reconstruct History, Restoration Specialist, Shreds of Sanity